dependent-usage-analyzer/index.js

```diff
@@ -126,6 +126,30 @@ function getComponentUsagesInFiles(files, rootDir) {
           addComponentUsage(fullComponentName, node.loc.start);
         }
       },
+      AssignmentExpression(node) {
```

Comment:
It might be helpful to add comments for each of these different types to describe their purpose for future us and other developers. For example, what does an `AssignmentExpression` represent? What about a `CallExpression`? Adding a couple comments in code could help clarify.

Reply:
Yes, I added for all methods in `walk.simple`